src/loading.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+require('material-design-lite/material.min.css');
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+class Loading extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    const { spinner } = props;
+
+    this.baseClasses = {
```

Comment:
The base classes should be determined on each render.  These components are eventually going to be migrated to pure function components in react 0.14 and keeping this in render will make that easier